src/Vocup/Forms/VocabularyBookSettings.cs

```diff
@@ -79,7 +80,7 @@ private void BtnOK_Click(object sender, EventArgs e)
     {
         book.MotherTongue = TbMotherTongue.Text;
         book.ForeignLang = TbForeignLang.Text;
-        book.PracticeMode = RbModeAskForeignLang.Checked ? PracticeMode.AskForForeignLang : PracticeMode.AskForMotherTongue;
+        book.PracticeMode = RbModeAskForeignLang.Checked ? PracticeMode.AskForForeignLang : RbModeAskMotherTongue.Checked ? PracticeMode.AskForMotherTongue : PracticeMode.AskForBothMixed;
```

Comment:
With more practice modes this line of code is hardly readable anymore. Maybe it would be clearer with a switch expression.